android/quest/src/main/assets/sync_by_organization_publisher.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "resourceType": "SearchParameter",
+    "id": "Organization-identifier",
+    "url": "http://hl7.org/fhir/SearchParameter/Organization-identifier",
+    "version": "4.0.1",
+    "name": "organization",
+    "code": "organization",
```

Comment:
can we remove extra and repeating and some of unused properties for now like url, version, id